### PR TITLE
Fixed the website header.

### PR DESCRIPTION
**Pull request by 2024-SANDHYA**
1. Added favicon to every page.
2. Fixed website header.
3. Added some hover effects
4. Added glowing effects
5. Added some text  & fonts in the hero section

### CODE REVIEW

**Comment by akshitadixit** [comment]
@2024-SANDHYA kindly resolve conflicts.

**Comment by 2024-SANDHYA** [comment]
> 
> 
> @2024-SANDHYA kindly resolve conflicts.

Resolved ma'am. Before merging you can check the changes here : https://2024-sandhya.github.io/Structurex/

File: index.html [hunk 1]
```diff
@@ -39,36 +39,35 @@
...
-     <a class="navbar-brand1" rel="home" href="#" title="logo">
-            <img src="src/assets/logo.png" width="90" height="80" >
-    <div class="container1 blue circleBehind">
-        <a href="#" id="menu-icon"></a>
-        <a href="#" class="current">Home</a>
-        <a href="sort.html">Sorting</a>
-        <a href="search.html">Searching</a>
-        <a href="#">Path Finding</a>
-        <a href="#">Graph</a>
-		<div class="dropdown">
```
Comment (akshitadixit) [inline index.html:51]: @2024-SANDHYA this dropdown menu was added in the PR that was last merged, you should not reject it.

File: search.html [hunk 2]
```diff
@@ -3,36 +3,35 @@
...
-                        <img src="src/assets/logo.png" width="90" height="80" >
-                <div class="container blue circleBehind">
-                    <a href="#" id="menu-icon"></a>
-                    <a href="index.html">Home</a>
-                    <a href="sort.html">Sorting</a>
-                    <a href="#" class="current">Searching</a>
-                    <a href="#">Path Finding</a>
-                    <a href="#">Graph</a>
-					<div class="dropdown">
-						<button onclick="show_hide()" class="Morebutton" > More </button>
```
Comment (akshitadixit) [inline search.html:27]: Same as previous comment.

File: sort.html [hunk 3]
```diff
@@ -3,7 +3,9 @@
     <head>
         <meta charset="UTF-8" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <title>Structurex</title>
```
Comment (akshitadixit) [inline sort.html:6]: Nice that you changed the title.

File: sort.html [hunk 4]
```diff
@@ -12,26 +14,18 @@
...
                 </a>
-                <div class="container blue circleBehind">
-                    <a href="#" id="menu-icon"></a>
-                    <a href="index.html" class="current">Home</a>
-                    <a href="#">Sorting</a>
-                    <a href="search.html">Searching</a>
-                    <a href="#">Path Finding</a>
-                    <a href="#">Graph</a>
-					<div class="dropdown">
-						<button onclick="show_hide()" class="Morebutton" > More </button>
```
Comment (akshitadixit) [inline sort.html:26]: Here.

File: search.html (hunk 2 above)
Comment (2024-SANDHYA) [reply search.html:27]: It was by mistake , I'll add it again , no issues.

File: sort.html (hunk 4 above)
Comment (2024-SANDHYA) [reply sort.html:26]: @akshitadixit  Done ma'am . Made all the requested changes.  🙂

**Comment by Fab-Gaurav** [comment]
LGTM 👍🏻

**Review by Ayushparikh-code — APPROVED** [review]

**Pull request merged**